py3status/py3.py

```diff
@@ -93,3 +98,33 @@ def replace_fn(match):
 
         format_string = re.sub('\{[^}]*\}', replace_fn, format_string)
         return format_string.format(**param_dict)
+
+    def check_commands(self, cmd_list):
+        """
+        Checks to see if commands in list are available using `which`.
+        Returns the first available command.
+        """
+        devnull = open(os.devnull, 'w')
+        for cmd in cmd_list:
+            c = shlex.split('which {}'.format(cmd))
+            if call(c, stdout=devnull, stderr=devnull) == 0:
+                return cmd
+
+    def play_sound(self, sound_file):
+        """
+        Plays sound_file if possible.
+        """
+        self.stop_sound()
+        cmd = self.check_commands(['paplay', 'play'])
+        if cmd:
+            sound_file = os.path.expanduser(sound_file)
+            c = shlex.split('{} {}'.format(cmd, sound_file))
+            self._audio = Popen(c)
```

Comment:
unless I'm mistaken, this still doesn't catch any output the command may throw. It is my understanding that you **need** to catch them using PIPES.

Example : https://github.com/ultrabug/py3status/blob/master/py3status/modules/xrandr.py#L111

Am I wrong ?